Mars_Application_UserStory2_TestAutomation/StepDefinitions/NegativeInvalidStepDefinitions.cs

```diff
@@ -0,0 +1,52 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using TechTalk.SpecFlow;
+using OpenQA.Selenium.Interactions;
+using System;
+using static System.Net.Mime.MediaTypeNames;
+using Mars_Application_UserStory2_TestAutomation.Pages;
+using TechTalk.SpecFlow.Assist;
+using Assert = Xunit.Assert;
+
+namespace Mars_Application_UserStory2_TestAutomation.StepDefinitions
+{
+    [Binding]
+    public class NegativeInvalidStepDefinitions
+    {
+        private readonly IWebDriver driver;
```

Comment:
please make use of constructor! fix this every where you have not used a constructor see 
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors